cnames_active.js

```diff
@@ -2654,6 +2654,7 @@ var cnames_active = {
   "riffy": "cname.vercel-dns.com", // noCF
   "riiickyy": "riiickyy.github.io",
   "riklewis": "riklewis.github.io",
+  "rilling": "samuelrilling.github.io/Rilling"
```

Comment:
As the CI error notes, your entry is missing a comma at the end of the line.

```suggestion
  "rilling": "samuelrilling.github.io/Rilling",
```

Reply:
Ok now i get it. Should be OK now. Sorry for the confusion